packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -477,6 +477,16 @@ export const prodParasKusama: EndpointOption[] = [
       Dwellir: 'wss://zeitgeist-rpc.dwellir.com',
       OnFinality: 'wss://zeitgeist.api.onfinality.io/public-ws'
     }
+  },
+  {
+    info: 'snow',
+    homepage: 'https://icenetwork.io/snow',
+    paraId: 2129,
+    text: 'Snow Network',
+    isUnreachable: true,
+    providers: {
+      IceNetwork: 'wss://snow-rpc.icenetwork.io',
```

Comment:
```suggestion
      IceNetwork: 'wss://snow-rpc.icenetwork.io'
```

Trailing comma removed.

It also needs to be alphabetical.